app-backend/tile/src/main/scala/LayerCache.scala

```diff
@@ -221,27 +221,29 @@ object LayerCache extends Config with LazyLogging with KamonTrace {
     toolRunId: UUID,
     subNode: Option[UUID],
     user: User,
-    voidCache: Boolean = false
+    colorRamp: ColorRamp
   ): OptionT[Future, ColorMap] = traceName("LayerCache.toolRunColorMap") {
-    val cacheKey = s"colormap-$toolRunId-${subNode}-${user.id}"
-    if (voidCache) rfCache.delete(cacheKey)
+    val cacheKey = s"colormap-$toolRunId-${subNode}-${user.id}-${colorRamp.getClass.getName}"
```

Comment:
Not sure what the key is coming out to -- but it seems a little too sticky and doesn't change with the query parameters.